Src/ReadGps.c

```diff
@@ -6,15 +6,94 @@
 
 #include "Data.h"
 
+
 static int READ_GPS_PERIOD = 1000;
+static int CMD_TIMEOUT = 1500;
+
+static uint8_t header = {0xB5, 0x62};
```

Comment:
header for what?

Reply:
Of all commands. I thought of at first having them desperately and then concatenating the strings but then decided against that. Will remove that code